src/components/CardDisplay/CardDisplay.tsx

```diff
@@ -0,0 +1,133 @@
+import { CardAddDisplay } from './CardAddDisplay';
+import { Box, Button, Grid, HStack, Typography, VStack } from '@/shared/components';
+import { styleToken } from '@/shared/styles';
+import { StyleProps } from '@/shared/types';
+import { replaceMaskText } from '@/shared/utils';
+import { CardState } from '@/type';
+
+type CardSize = 'big' | 'small';
+type CardTypographyVariant = 'headline' | 'body';
+type CardDisplayProps = {
+  size: CardSize;
+  onClick?: () => void;
+} & Pick<CardState, 'label' | 'color' | 'cardNumber' | 'expirationDate' | 'ownerName'>;
+
+export const CardDisplay = ({
+  size,
+  label,
+  color = styleToken.color.body,
+  cardNumber,
+  expirationDate,
+  ownerName,
+  onClick,
+}: CardDisplayProps) => {
+  const { cardDisplayProps, cardChipProps, typographyVariant, maskFontSize } = getCardStyles(size);
+  const [expirationMonth, expirationYear] = expirationDate;
+  return (
+    <Button onClick={onClick} padding="0">
+      <VStack
+        color={styleToken.color.gray600}
+        backgroundColor={color}
+        boxShadow={`3px 3px 5px ${styleToken.color.shadow}`}
+        borderRadius="5px"
+        padding="10px 14px"
+        spacing="0"
+        {...cardDisplayProps}
+      >
+        <Box width="100%" textAlign="left">
+          <Box height="24px">{label && <Typography variant="body">{label}</Typography>}</Box>
+          <Box backgroundColor={styleToken.color.mustard} borderRadius="4px" {...cardChipProps} />
+        </Box>
+        <VStack alignItems="center" justifyContent="space-between" width="100%" flexGrow={1}>
+          <Box width="100%" />
+          <CardNumberDisplay cardNumber={cardNumber} variant={typographyVariant} maskFontSize={maskFontSize} />
+          <CardOwnerDisplay
+            ownerName={ownerName}
+            variant={typographyVariant}
+            expirationMonth={expirationMonth || 'MM'}
+            expirationYear={expirationYear || 'YY'}
```

Comment:
```suggestion
            expiration={expirationDate}
```
이렇게 표현해주고 내부로 위임하면 어떨까 싶기도 하고..

Reply:
> 이렇게 표현해주고 내부로 위임하면 어떨까 싶기도 하고..

변경되지 않을 형식이라 생각되어
컴포넌트 내부에서 defaultValue 설정을 해주는게 올바르게 느껴집니다.